_posts/2020-11-24-Multiple-Audio-Interfaces.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "Using Multiple Audio Interfaces"
+heading: "Using multiple Audio interfaces"
+author: "gilgongo"
```

Comment:
I'm not actually the author, I just edited it a bit. I must have not credited the person who wrote it (I think it was originally on the forums). That's a bit bad.

Reply:
Ok. So we can either remove it or add the correct author (we might need to add it to the authors.yml file too)